app/discussion/server/methods/createDiscussion.js

```diff
@@ -33,7 +33,7 @@ const mentionMessage = (rid, { _id, username, name }, message_embedded) => {
 	return Messages.insert(welcomeMessage);
 };
 
-const create = ({ prid, pmid, t_name, reply, users }) => {
+const create = ({ prid, pmid, t_name, reply, t, users }) => {
```

Comment:
Discuss with the team and re-evaluate, weather to implement comments this way.
(allowing public child of a private parent room)